frontend/src/concepts/pipelines/content/tables/utils.ts

```diff
@@ -22,7 +22,7 @@ export const getRunDuration = (run: PipelineRunKF): number => {
 };
 
 export const getStatusWeight = (run: PipelineRunKF): number => {
-  const weights: Record<PipelineRunStatusesKF, number> = {
+  const weights: Record<PipelineRunStatusesKF, number | undefined> = {
```

Comment:
```suggestion
  const weights: Record<PipelineRunStatusesKF | string, number | undefined> = {
```

Then change the return statement to:
```
return weights[run.status] ?? Infinity;
```